packages/sdk/src/browser-wallets/unisat/index.ts

```diff
@@ -8,11 +17,94 @@ function isInstalled() {
   return typeof window.unisat !== "undefined";
 }
 
-async function getAddresses() {}
+async function getAddresses(network: Network): Promise<WalletAddress[]> {
+  if (!isInstalled()) {
+    throw new OrditSDKError("Unisat not installed.");
+  }
+
+  if (!network) {
+    throw new OrditSDKError("Invalid options provided.");
+  }
+
+  const targetNetwork = NETWORK_TO_UNISAT_NETWORK[network];
+  if (!targetNetwork) {
+    throw new OrditSDKError("Unsupported network.");
+  }
+
+  const connectedNetwork = await window.unisat.getNetwork();
+  if (connectedNetwork !== targetNetwork) {
+    await window.unisat.switchNetwork(targetNetwork);
+  }
+
+  const accounts = await window.unisat.requestAccounts();
+  const publicKey = await window.unisat.getPublicKey();
+
+  const address = accounts[0];
+  if (!address) {
+    return [];
+  }
+  const format = getAddressFormat(address, network);
+  return [
+    {
+      pub: publicKey,
+      address,
+      format,
+    },
+  ];
+}
+
+async function signPsbt(
+  psbt: Psbt,
+  options: UnisatSignPSBTOptions = {},
```

Comment:
we need to put `options` as `options?` , because if not then we always have to pass second param even as `{}` . 🤔 

would suggest to use named params instead but lets not have big breaking change for now

Reply:
parameter cannot have question mark and initializer in the same line. I've fixed this issue by making options optional in the `BrowserWallet` interface